src/Business/Detection/Fetching/Models/MTimestampFilter.cs

```diff
@@ -0,0 +1,11 @@
+namespace Business.Detection.Fetching.Models
+{
+    using System;
+
+    public class MTimestampFilter
+    {
+        public DateTime LesserThan { get; set; }
```

Comment:
Should it be a nullable date time instead?

Reply:
done